decidim-process-extended/app/decorators/decidim/participatory_processes/admin/participatory_processes_controller_decorator.rb

```diff
@@ -0,0 +1,120 @@
+# frozen_string_literal: true
+
+Decidim::ParticipatoryProcesses::Admin::ParticipatoryProcessesController.class_eval do
+  def update
+    enforce_permission_to :update, :process, process: current_participatory_process
+    @form = form(Decidim::ParticipatoryProcesses::Admin::ParticipatoryProcessForm).from_params(
+      participatory_process_params,
+      process_id: current_participatory_process.id
+    )
+
+    Decidim::ParticipatoryProcesses::Admin::UpdateParticipatoryProcess.call(current_participatory_process, @form) do
+      on(:ok) do |participatory_process|
+
+        # add "default" hero / banner image code
+        if participatory_process.hero_image.url.blank?
+          add_default_image_hero (participatory_process.id)
+          add_default_image_banner (participatory_process.id)
+        end
+
+        flash[:notice] = I18n.t("participatory_processes.update.success", scope: "decidim.admin")
+        redirect_to edit_participatory_process_path(participatory_process)
+      end
+
+      on(:invalid) do
+        flash.now[:alert] = I18n.t("participatory_processes.update.error", scope: "decidim.admin")
+        render :edit, layout: "decidim/admin/participatory_process"
+      end
+    end
+  end
+
+  private
+
+  def ability_context
+    super.merge(current_participatory_space: current_participatory_process)
+  end
```

Comment:
This method is not used anymore in Decidim, they were renamed to `permissions_context` in the permissions rework.

But, anyway, we can remove it because Decidim already have this:

The controller is this and includes the `ParticipatorySpaceAdminContext` concern: https://github.com/decidim/decidim/blob/develop/decidim-participatory_processes/app/controllers/decidim/participatory_processes/admin/participatory_processes_controller.rb#L10 

The concern have this method:https://github.com/decidim/decidim/blob/35305653823fc5241ac08b2cf99f12061aad8ca3/decidim-admin/app/controllers/concerns/decidim/admin/participatory_space_admin_context.rb#L50-L54

```suggestion
```

Reply:
Hi @rbngzlv !
yeah! good catch here! I reviewed permissions rework "Fine grained permissions #3029" PR and as you pointed out, they removed it on commit:
https://github.com/decidim/decidim/commit/dc1e00d0ff6921fa869c47cdeee512f3344ce975#diff-d5d7e3490961c4ba9b608fa2eac51bf4
in favor of the concern you `ParticipatorySpaceAdminContext` as you said. :smile: 
So, I removed it from here too.
Thanks!! :pray: